lte/gateway/python/integ_tests/s1aptests/test_attach_detach_cloud.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+import unittest
+
+import s1ap_types
+from integ_tests.s1aptests import s1ap_wrapper
+
+
+class TestAttachDetachCloud(unittest.TestCase):
+
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+
+    def test_attach_detach_cloud(self):
+        """ Basic attach/detach test with a single UE """
+        num_ues = 1
```

Comment:
Can we configure num_ues to 2, as detach_type and wait_for_s1 are used set to different values for each UE